frontend/src/components/layout/Header/EmptyHeader.tsx

```diff
@@ -0,0 +1,45 @@
+import './Header.scss';
```

Comment:
this header is displayed when lazy loading components. Makes the application appear consistent during loads.